src/SourceGenerators/Uno.UI.SourceGenerators.netcore.Tests/MetadataUpdateTests/Given_HotReloadService.cs

```diff
@@ -55,22 +62,23 @@ public record DiagnosticsResult(string Id);
 			var scenarioName = Path.GetFileName(scenarioFolder);
 			var path = Path.Combine(scenarioFolder, "Scenario.json");
 
-			//if (scenarioName != "When_Simple_Xaml_ResourceDictionary_Change_One")
+			//if (scenarioName != "When_Two_Projects_Single_Code_File_With_Code_Update")
 			//{
 			//	continue;
 			//}
```

Comment:
Thi commented part is still needed here?

Reply:
Yes, it's useful because tests are not individual methods, so it avoids running multiple tests unnecessarily during debugging.

Reply:
Thanks for the details @jeromelaban :)